src/Desktop.Analyzers/Desktop.Analyzers.md

```diff
@@ -48,6 +48,16 @@ Severity: Warning
 
 Help: [http://msdn.microsoft.com/library/ms182350.aspx](http://msdn.microsoft.com/library/ms182350.aspx)
```

Comment:
You mean editing rule description? If so, I disagree, because the purpose of the string is not for creating markdown file. That being said, if you are suggesting to use quote for all the types in description, I'm open to discussion :)